.bundle/config

```diff
@@ -0,0 +1,2 @@
+---
+BUNDLE_PATH: "vendor/bundle"
```

Comment:
This is quite an interesting change. I wasn't experiencing the problem at all, because I don't use the system Ruby on macOS. Regardless of whether we go ahead with the change, I recommend using something like [rbenv](https://github.com/rbenv/rbenv). It removes a lot of headaches with Ruby versions and gems.

However, I agree that everything should work out of the box regardless of the environment and this seems to be a good way to do that.

Reply:
Agreed re: `rbenv`(!!!), but like you said, working out of the box is a nice goal, partially b/c it makes the on-ramp for contributing that much easier! :) 

This change has the same effect as https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/Rakefile#L48, but because we don't use a `rakefile` for setting up the repo, doing it manually is the next best thing (probably?)

What's nice about doing it this way is that if someone isn't following the directions and is just trying to figure it out, `bundle` will do the right thing for them here.